README.md

```diff
@@ -1,7 +1,7 @@
 # Go stats and tracing collection libraries
 
 [![Build
-Status](https://travis-ci.org/census-instrumentation/opencensus-go.svg?branch=master)](https://travis-ci.org/census-instrumentation/opencensus-go)
+Status](https://travis-ci.org/census-instrumentation/opencensus-go.svg?branch=master)](https://travis-ci.org/census-instrumentation/opencensus-go) [![GoDoc](https://godoc.org/github.com/census-instrumentation/opencensus-go?status.svg)](https://godoc.org/github.com/census-instrumentation/opencensus-go)
```

Comment:
If you don't mind I like using labels here (look at the raw comment):

[![Build Status][travis-image]][travis-url] [![GoDoc][godoc-image]][godoc-url]

[travis-image]: https://travis-ci.org/census-instrumentation/opencensus-go.svg?branch=master
[travis-url]: https://travis-ci.org/census-instrumentation/opencensus-go
[godoc-image]: https://godoc.org/github.com/census-instrumentation/opencensus-go?status.svg
[godoc-url]: https://godoc.org/github.com/census-instrumentation/opencensus-go